pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/SyntheticStageNames.groovy

```diff
@@ -28,22 +28,26 @@ package org.jenkinsci.plugins.pipeline.modeldefinition
 
 public class SyntheticStageNames {
     public static String checkout() {
-        return "Checkout SCM"
+        return "Declarative: Checkout SCM"
```

Comment:
Good point, but I really do want to make sure we have uniqueness for the synthetic stages. Any ideas?